app/src/main/java/com/cornellappdev/android/volume/ui/viewmodels/FlyersViewModel.kt

```diff
@@ -117,9 +123,11 @@ class FlyersViewModel @Inject constructor(
                 flyersUiState =
                     flyersUiState.copy(upcomingFlyersState = FlyersRetrievalState.Loading)
                 if (categorySlug.lowercase() == "all") {
+                    val initialFlyers = flyerRepository.fetchFlyersAfterDate(getToday())
+                    Log.d(TAG, "queryUpcomingFlyers: $initialFlyers")
```

Comment:
do you still need this log statement and all the other log statements in this file? if not, please delete them